orders/models.py

```diff
@@ -0,0 +1,20 @@
+from django.db  import models
+from utils      import TimeStampModel
+
+class OrderItem(models.Model):
+    product_group    = models.ForeignKey("products.ProductGroup", on_delete = models.CASCADE)
+    order            = models.ForeignKey("Order", on_delete = models.CASCADE)
+    quantity         = models.PositiveIntegerField()
+
+    class Meta:
+        db_table = "order_items"
+
+class Order(TimeStampModel):
+    user             = models.ForeignKey("users.User", on_delete = models.CASCADE)
+    user_coupon      = models.ForeignKey("users.UserCoupon", on_delete = models.CASCADE)
+    delivery_price   = models.PositiveIntegerField()
+    status           = models.CharField(max_length = 120)
```

Comment:
`status`를 table로 정규화해주세요~
```suggestion
    status           = models.CharField(max_length = 120)
```

Reply:
status 테이블 정규화해서 진행했습니다 !